recipes/google-weather-tools/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "google-weather-tools" %}
+{% set version = "0.3.2" %}
+{% set git_rev = "conda-candidate-v0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/google/weather-tools.git
+  git_rev: "{{ git_rev }}"
+
```

Comment:
We generally only want to be using released versions.

Reply:
+1  - is there a way to configure this to reflect our git release tags?